egs2/libritts_r/enh1/db.sh

```
@@ -0,0 +1,27 @@
+# egs2/libritts_r/enh1/db.sh
+# Dataset and noise source paths for the ESPnet-Sidon recipe.
+# Set paths to existing directories; leave "" to skip.
+#
+# SR information is NOT stored here — it lives in local/data.sh.
+# To add a new dataset: add DATASET_* here + SR entry in local/data.sh.
+
+# ── Clean speech datasets ─────────────────────────────────────────────────
+DATASET_LIBRITTS_R=""   # 585h, 24kHz, en  (mandatory)
```

Shellcheck (0.11.0)

[warning] DATASET_LIBRITTS_R appears unused. Verify use (or export if used externally).

(SC2034)